lib/cn/kvset.c

```diff
@@ -3639,6 +3692,121 @@ kvset_minkey(struct kvset *ks, const void **minkey, u16 *minklen)
     *minklen = ks->ks_minklen;
 }
 
+struct vgmap *
+vgmap_alloc(uint32_t nvgroups)
+{
+    struct vgmap *vgmap;
+    const size_t sz = sizeof(*vgmap) + nvgroups *
+        (sizeof(*(vgmap->vbidx_out)) + sizeof(*(vgmap->vbidx_adj)) + sizeof(*(vgmap->vbidx_src)));
+
+    vgmap = calloc(1, sz);
```

Comment:
Seems like the calloc is unnecessary and this could just be a malloc, since you are going to fill every index in here anyways.

Reply:
Lemme check and update that.